src/objective-c/change-comments.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python
```

Comment:
Nathaniel - Changed, plus added copyright notice and a short description of what the script does.